CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 2.24.0-rc.0 / 2020-12-30
+
+* [FEATURE] Add TLS and basic authentication to HTTP endpoints. #8316
+* [FEATURE] promtool: Add `check web-config` command to check web config files. #8319
```

Comment:
Don't we call these subcommands?

Reply:
I was on the fence of calling them that way. As you prefer it, I'm happy to change it.